winbuild/build.rst

```diff
@@ -42,8 +42,8 @@ behaviour of ``build_prepare.py``:
   If ``PYTHON`` is unset, the version of Python used to run
   ``build_prepare.py`` will be used. If only ``PYTHON`` is set,
   ``EXECUTABLE`` defaults to ``python.exe``.
-* ``ARCHITECTURE`` is used to select a ``x86`` or ``x64`` build. By default,
-  uses same architecture as the version of Python used to run ``build_prepare.py``.
+* ``ARCHITECTURE`` is used to select a ``x86``, ``x64`` or ``ARM64``build.
+  By default, uses same architecture as the version of Python used to run ``build_prepare.py``.
   is used.
```

Comment:
This change seems to have disappered in a force-push. Is there any reason not to include it?

```suggestion
  is used. Note that the ``ARM64`` build is not fully supported and some dependencies may fail to build. Pull requests are welcome.
```

Reply:
Sorry I didn't mention that LittleCMS is the last dependency and VS2019 update is merged, just not released yet:
https://github.com/mm2/Little-CMS/pull/288
Patch to use VS2019 project: https://github.com/gaborkertesz-linaro/Pillow/commit/0af6042c5c0a412c2614147458ce2d4832436bc8

So if either VS2019 project is accepted or VS2017 will be updated for ARM64 as well and LCMS2 is released, all external dependencies will be enabled.

Reply:
lcms2 2.13 has now been released.

Reply:
Thanks for the heads up, I'm going to rebase my changes and run the tests!